libentitlement.go

```diff
@@ -31,8 +32,22 @@ func NewEntitlementsManager(profile *secprofile.Profile) *EntitlementsManager {
 }
 
 // GetProfile() returns the current state of the security profile
-func (m *EntitlementsManager) GetProfile() *secprofile.Profile {
-	return m.profile
+func (m *EntitlementsManager) GetProfile() (*secprofile.Profile, error) {
+	if m.profile == nil {
+		return nil, fmt.Errorf("Entitlements Manager has np security profile.")
+	}
+
+	return m.profile, nil
+}
+
+// SetProfile() sets the entitlement manager's security profile
```

Comment:
hm ok.  I was wondering if the output of 4. should be setting the entitlement manager to a "done" state and outputting a profile/some other spec.

We'll figure out what makes most sense after Moby/K8s integrations, so no need to over-specify now.